Cloud_Providers.md

```diff
@@ -0,0 +1,50 @@
+# Monitoring and Logging on Cloud Provider Platforms
+
+This document contains information about using the SAS Viya logging and monitoring components in cloud provider platforms (Azure, Amazon, and Google) that are supported by SAS Viya.
+
+The logging and monitoring applications provided in this repository for SAS Viya can be deployed and can run in any of the cloud provider's environments. Minimal customization is needed to deploy these applications. Because the applications provided for SAS Viya work the same way across cloud platforms, administrators can have a consistent experience when moving between cloud platforms. 
+
+The configurations for the monitoring and logging applications provided for SAS Viya are focused on detailed metrics and messages from the applications and services that make up SAS Viya. In some cases, the monitoring application provided by a cloud provider does not support the same level of detail. For administrators who are interested in metrics and log messages from SAS Viya services and components, the applications included in this repository provide the most comprehensive view of this detailed information. For administrators who are primarily interested in information from cluster resources, native applications might provide the level of logging and monitoring support that is needed.
+
+Although each supported cloud provider provides one or more native applications to display metrics and log messages, configuration and customization steps are required in order to integrate collected SAS Viya metric data and log messages with these applications. Samples are included in this repository to provide the steps required to display SAS Viya metrics and log messages on these applications. However, not all scenarios are currently supported.   
+
+## Microsoft Azure
+
+### Deploying SAS Viya Logging and Monitoring on Azure
+
+In order to deploy SAS Viya logging and monitoring on Azure, you must create a custom Storage Class. 
+Follow the procedures in the [Azure sample](/samples/azure-deployment/README.md) for information on customizing your deployment for Azure.
+
+You must also deploy logging and monitoring using ingress.
+
```

Comment:
Maybe I'm being too picky, but the statement that you must create a default storage class is a bit too strong. Basically, the default storage class is not suitable. A solution to that is to create a custom storage class, but it could also be to use some other existing storage class. I'm not sure if we should make that subtle distinction, but that's the core issue.

Reply:
Added a mention of the ingress sample and the fact that ingress is covered in the Azure sample. Clarified the information about the StorageClass.

Reply:
Works for me.